torch/csrc/api/include/torch/nn/options/activation.h

```diff
@@ -56,5 +57,20 @@ struct LeakyReLUOptions {
   TORCH_ARG(bool, inplace) = false;
 };
 
+// ============================================================================
+
+/// Options for the Softmax functional and module.
+struct TORCH_API SoftmaxOptions {
+  SoftmaxOptions(int dim = -1, torch::Dtype dtype = torch::Dtype::Undefined);
```

Comment:
I think it should be
```suggestion
  SoftmaxOptions(int dim);
```
because `dim` shouldn't be an optional argument.

Reply:
Thanks for your suggestion. I fixed it.